### DISCUSSION TITLE
Calculation time

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1508] [General]
Is it normal to take 2 hours and more for 13-15 SCC cycles with a SCC Tolerance of 1E-5 for a system of 999 atoms in it with 8 cpus and 8 threads? It was the best setting that reduced the wall clock time for other systems with similar size. 
Can anyone please recommend the best setting for parallel usage of DFTB+? for normally a system of ~1000 atoms?

Thank you!

**Comment by aradi** [comment]
No, it is not normal. Using the non-mpi conda version of DFTB+ 24.1, an 1024 atom periodic SiC takes less than 3 minutes for 7 SCC steps on my 7 years old laptop using 2 threads.

Please consult a local expert for your computing system, I do not think that we can help from remote. Make sure, to use only a low number of threads first and increase it. If you use the MPI-version of DFTB+, make sure, to set the nr. of threads to 1.
